docs/dev-env-first-time-contributors.md

```diff
@@ -711,7 +711,30 @@ This is equivalent of running a halt followed by an up (aka rebooting
 the guest).  After this, you can do `vagrant provision` and `vagrant
 ssh`.
 
-#### ssh connection closed by remote host
```

Comment:
@KiranShahdadpuri It appears that you accidentally removed this line: "#### ssh connection closed by remote host", am I correct?